package.json

```diff
@@ -29,6 +30,7 @@
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.4.2",
     "react-toastify": "^9.1.1",
+    "swr": "^2.2.2",
```

Comment:
i was thinking of using react-query, do you know it?

Reply:
I can look into it.